### PR TITLE
Add SVSE2 tournament page

### PR DESCRIPTION
**Pull request by WalterToro**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/SVSE/2/en.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+
+| Player A |  |  | Player B | Match Link |
+| --: | :-: | :-: | :-- | :-- |
+| Kamikho ::{ flag=PE }:: | 0 | **3** | ::{ flag=VN }:: **mightyxd** | [#1](https://osu.ppy.sh/community/matches/109770469) |
+| ERA Punish ::{ flag=DE }:: | 0 | **3** | ::{ flag=VN }:: **Lott** | [#1](https://osu.ppy.sh/community/matches/109770747) |
+
+### Week 4
+
+Teams with their name on bold are advance to finals.
+Teams with their name crossed out are eliminated.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:166]:
```suggestion
Teams with emboldened names advance to finals.
Teams with names crossed out are eliminated.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+| #7 | Makuron de misshon | 4 | 9 | 6.5 | 1,966,813 |
+| #8 | ~~:yangu:~~ | 3 | 10 | 6.5 | 1,963,982 |
+| #9 | ~~Billy et son chien~~ | 10 | 4 | 7 | 1,954,437 |
+| #10 | ~~kouvola mcdonalds meetup~~ | 9 | 8 | 8.5 | 1,960,057 |
+| #11 | ~~broke on mems~~ | 11 | 11 | 11 | 955,101 |
+
+### Week 3
+
+Teams with their name on italics are on last chance situation.
+Teams with their name crossed out are eliminated.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:185]:
```suggestion
Teams with italicised names are on last chance situation.
Teams with names crossed out are eliminated.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+| #14 | ~~Jouœurs rincés~~ | 9 | 18 | 13.5 | 1,687,143 |
+| #15 | ~~De Grachtengordelse Schavuiten~~ | 17 | 13 | 15 | 1,878,081 |
+| #16 | ~~team a~~ | 14 | 16 | 15 | 1,789,481 |
+| #17 | ~~Team at~~ | 18 | 14 | 16 | 1,866,262 |
+| #18 | ~~Taiko Main forced into SV~~ | 16 | 17 | 16.5 | 1,715,982 |
+
+### Week 2
+
+Teams with their name on italics are on last chance situation.
+Teams with their name crossed out are eliminated.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:211]:
```suggestion
Teams with italicised names are on last chance situation.
Teams with names crossed out are eliminated.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+| #27 | ~~sata andagi!~~ | 24 | 25 | 24.5 | 1,884,784 |
+| #28 | ~~north sentinel island~~ | 30 | 23 | 26.5 | 1,872,987 |
+| #29 | ~~:1217angry:~~ | 27 | 29 | 28 | 1,682,550 |
+| #30 | ~~3 tortues trotaient sur 1 toit~~ | 29 | 30 | 29.5 | 1,387,927 |
+| #31 | ~~Schizo Velocity~~ | 31 | 32 | 31.5 | 895,544 |
+| #32 | ~~cta lovers~~ | 32 | 32 | 32 | 0 |
+
+### Week 1
+
+Teams with their name on italics are on last chance situation.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:250]:
```suggestion
Teams with italicised names are on last chance situation
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+| #41 | Beans On Toast | 3,303,355 |
+| #42 | Forniteloliclub | 3,050,612 |
+| #43 | SVn't | 2,705,243 |
+| #44 | Losmatapro | 2,529,694 |
+
+## Ruleset
+
+### Overall notes
+
+1. SV: Summer Exhibition 2 is a **team tournament**, played on the osu!mania game mode.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:342]:
```suggestion
1. SV: Summer Exhibition 2 is an osu!mania **team tournament**.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+  - **Qualifiers and Final stage**: Mappool will be released one week in advance, right after the mappool showcase.
+  - **Weekly stages**: Mappool will be released every Friday at 0 UTC.
+6. Maps will be played with [FreeMod](/wiki/Gameplay/Game_modifier#freemod). The only allowed mods are FadeIn, Flashlight, Hidden, and Mirror.
+
+### Tournament structure
+
+#### Qualifiers
+
+1. The Qualifier pool contains 4 maps, all of which will use [FreeMod](/wiki/Gameplay/Game_modifier#freemod) rules.
+2. Teams will select one player to play each map and each player will play two different maps from the mappool.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:360]: this reads convoluted, any chance you could rephrase/simplify it?

File: wiki/Tournaments/SVSE/2/en.md [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+3. When joining the lobby they will tell the referee what maps each player will be playing.
+  - Players cant change what maps are they going to play after this.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:362]:
```suggestion
3. When joining the lobby, the teams will tell the referee what maps each player will be playing.
  - From this point, players can't change selected maps.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 8]
```diff
@@ -0,0 +1,399 @@
...
+1. There will be 4 weeks of match play.
+2. Every week 2 maps will be released, one for "Side A" and another for "Side B".
+3. An Information Compass will be released for each of these maps every Monday at 0 UTC.
+4. After the Information Compasses are released, teams will sign up one player to the Side A lobby, and the other for the Side B lobby.
+  - They can do this until Thursday 23:59 UTC.
+5. On Friday at 0 UTC the maps are uploaded and posted in the server.
+6. Each player will have three tries at each map, their best score will count for each weekly stage.
+7. Each map will have a leaderboard with the best recorded scores on it, and their placement on those will be used for the overall week placement
+8. Teams advance to the next week stage if they are not eliminated, elimination happens if a team fulfills one of the folowing conditions:
+  - They get 2 "Last Chance" flags, these flags are given by being on the bottom half of the leaderboard each week.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:378]:
```suggestion
  - They get 2 "Last Chance" flags, which are given to the teams in the bottom half of the leaderboard each week.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 9]
```diff
@@ -0,0 +1,399 @@
...
+  - They are one of the bottom 4 teams on the leaderboard on a given week.
+9. Top 4 teams on the leaderboard by week 4 advance to finals.
+  - Teams outside of top 4 that dont get eliminated by the end of week 4 are survivors and get a small prize.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:381]:
```suggestion
  - Teams outside of top 4 not eliminated by the end of week 4 are considered survivors and get a small prize.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 10]
```diff
@@ -0,0 +1,399 @@
...
+1. Teams that advanced to finals will play a small single-elimination bracket.
+1. The Finals pool contains 5 maps, all of which will use [FreeMod](/wiki/Gameplay/Game_modifier#freemod) rules.
+2. Matches are defined as follows:
+  - First place faces Fourth place.
+  - Second place faces Third place.
+  - Losers of each match will define the 3rd and 4th places of the tournament.
+  - Winners of each match will define the Grand Champion of SVSE2.
+3. Teams needs to win 2 sets to win a match, to win a set they need to win 3 maps (best of 5).
+4. Teams will select an "A player" and a "B player", the A player of each team will face each other, similar to the B player on a 1 versus 1 set. These will be the first and second set of the match.
+  - These A and B players cant be swapped for the entire stage.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md:394]:
```suggestion
  - These A and B players can't be swapped for the entire stage.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 11]
```diff
@@ -0,0 +1,399 @@
...
+5. If both teams tie on set points by the end of the second set, a third set will be played, this time a 2 versus 2.
+6. Each team must use `!roll` command at the start of every set in `#multiplayer`.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md]:
```suggestion
6. Each team must use the `!roll` command at the start of every set in `#multiplayer`.
```

File: wiki/Tournaments/SVSE/2/en.md [hunk 12]
```diff
@@ -0,0 +1,399 @@
...
+   - The winner of the `!roll` starts picking first.
+   - The loser of the `!roll` picks second.
+7. There wont be bans in this stage.
```
Comment (TicClick) [inline wiki/Tournaments/SVSE/2/en.md]:
```suggestion
7. There will be no bans in this stage.
```

**Review by TicClick — CHANGES_REQUESTED** [review]

**Comment by agatemosu** [comment]
Identation should be 3 i think

**Comment by WalterToro** [comment]
> Identation should be 3 i think

Thank you!

**Review by TicClick — APPROVED** [review]

**Pull request merged**